src/k8-config/src/context.rs

```diff
@@ -350,8 +362,26 @@ kubectl config use-context {{ name }}
 
         debug!("script {}", render);
 
-        let output = Command::new(tmp_file).output().expect("cluster command");
+        let output = Command::new(tmp_file)
+            .debug()
+            .output()
+            .expect("cluster command");
         io::stdout().write_all(&output.stdout).unwrap();
         io::stderr().write_all(&output.stderr).unwrap();
     }
 }
+
+mod cmd_ext {
+    use std::fmt::Debug;
+    use std::process::Command;
+    use tracing::debug;
+
+    pub trait CmdExt: Debug {
```

Comment:
shouldn't this be using flv-util?  or add to flv-util it self?

Reply:
Right, so the reason I didn't do that here is that this repo is a general-purpose kubernetes crate and I didn't want to add any dependencies to the Fluvio project itself, since I think making a habit of that will make things messier.

Reply:
we can split out Command related as separate crate from `flv-util` later so it can be used in other projects.   Can you create issue on `flv-util` to do that?  For now, it's ok